plugins/processors/awsapplicationsignals/processor.go

```diff
@@ -137,7 +138,10 @@ func (ap *awsapplicationsignalsprocessor) processMetrics(ctx context.Context, md
 			metrics := ils.Metrics()
 			for k := 0; k < metrics.Len(); k++ {
 				m := metrics.At(k)
-				m.SetName(metricCaser.String(m.Name())) // Ensure metric name is in sentence case
+				// Check if the first letter of the metric name is not capitalized
+				if len(m.Name()) > 0 && !unicode.IsUpper(rune(m.Name()[0])) {
```

Comment:
will there be documentation for custom metrics to ensure the first letter is not capitalized?

Reply:
It's a good question. I personally prefer respecting the original metric name...

I don't know if there will be documentation for naming convention, perhaps we should revisit this logic whether it makes sense to change the letter case.